poketrainer/poketrainer.py

```diff
@@ -286,8 +286,7 @@ def _heartbeat(self, res=False, login_response=False):
         if 'GET_INVENTORY' in responses:
 
             # update objects
-            inventory_items = responses.get('GET_INVENTORY', {}).get('inventory_delta', {}).get('inventory_items', [])
-            self.inventory = Inventory(self, inventory_items)
+            self.inventory.update_player_inventory()
```

Comment:
If I'm reading `Inventory.update_player_inventory()` correctly, wouldn't this change result in an extra `get_inventory` API call? In the existing implementation, it looks like we're just extracting data out of the previous API call's response.